src/cassandra/cli/load.cdl

```diff
@@ -0,0 +1,180 @@
+-- this cdl schema create the main tables for blueflood in cassandra using the cqlsh client
+-- usage:
+--   $ cqlsh hostname -f load.cdl -u username -p password
+-- example:
+--   $ cqlsh 127.0.0.1 -f ./blueflood/src/cassandra/cli/load.cdl
+
+CREATE KEYSPACE IF NOT EXISTS "DATA"
+    WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
+
+
+CREATE TABLE IF NOT EXISTS "DATA".metrics_locator (
+    key bigint,
+    column1 text,
+    value text,
+    PRIMARY KEY (key, column1)
+) WITH COMPACT STORAGE AND speculative_retry = 'NONE';
```

Comment:
no if they're different, but the two settings above "WITH COMPACT STORAGE AND speculative_retry = 'NONE';" just make sure the database file is small and no unnecessary cpu cycle attempting to retry db transactions on failure which is only useful in actual multi-nodes cluster (and not single node cluster).